### PR TITLE
Make README formatting more consistent

### PR DESCRIPTION
**Pull request by WillSewell**
The formatting is fairly messed up when GH renders it, so I had a go at tidying it up and making it more consistent.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,20 +41,18 @@ package main
...
-	}
-
-	data := map[string]string{"message": "hello world"}
-
-	// trigger an event on a channel, along with a data payload
-	client.Trigger("test_channel", "my_event", data)
+    data := map[string]string{"message": "hello world"}
 
+    // trigger an event on a channel, along with a data payload
+    client.Trigger("test_channel", "my_event", data)
```
Comment (kn100) [inline README.md]: Maybe change this to my_channel - this is the one the debug console subscribes to

File: README.md [hunk 2]
```diff
@@ -78,6 +76,7 @@ client := pusher.Client{
 ```go
 client := pusher.ClientFromURL("http://key:secret@api.pusherapp.com/apps/app_id")
```
Comment (kn100) [inline README.md]: Maybe change this so it's an api url that includes a cluster (ie api-mt1 or whatever)

File: README.md [hunk 3]
```diff
@@ -130,7 +128,7 @@ By default, this is `"api.pusherapp.com"`.
 
 Setting the `pusher.Client`'s `Cluster` property will make sure requests are sent to the cluster where you created your app.
 
-*NOTE! `Host` option has precendece over the `Cluster`, ie. if `Host` is set then `Cluster` will be ignored.
+*NOTE! `Host` option has precedence over the `Cluster`, ie. if `Host` is set then `Cluster` will be ignored.
```
Comment (kn100) [inline README.md]: I think this can probably just be simplified to the ie bit.

File: README.md [hunk 4]
```diff
@@ -144,32 +142,31 @@ As of version 1.0.0, this library is compatible with Google App Engine's urlfetc
...
-
-	client := pusher.Client{
-		AppId:  "app_id",
-		Key:    "key",
-		Secret: "secret",
-		HttpClient: urlfetchClient,
-	}
-
-	client.Trigger("test_channel", "my_event", map[string]string{"message": "hello world"})
+    client.Trigger("test_channel", "my_event", map[string]string{"message": "hello world"})
```
Comment (kn100) [inline README.md]: my_channel

File: README.md (hunk 1 above)
Comment (WillSewell) [reply README.md]: It's actually `my-channel`, but yes I agree that is a worthwhile improvement. I'll make the change!

**Comment by WillSewell** [comment]
Merging because I addressed your comments.

**Pull request merged**